Bff proposal additional improvements (#1)
andreaTP/architecture: _adr/92/index.adoc

* Potential lack of control over APIs being aggregated
* The introduction of private data planes can introduce additional issues because of backend APIs reachability

For private data plates, we know it is a known feature coming but its impact will not be done as part of this work.

> It could lead to change int he BFF pattern approached described here and to the control plane at large.

Reviewer: ```suggestion
It could lead to change in the BFF pattern approached described here and to the control plane at large.
```
